utils/fields.py

```diff
@@ -83,9 +89,22 @@ def generate_fake_data(self, faker, **kwargs):
         :param kwargs:
         :return: list
         """
-        num_items = randint(self.min_items, self.max_items)
         fake_data = []
 
+        if (overrides := kwargs.get("overrides")) and self.name in overrides:
```

Comment:
nice syntactic sugar! I've never seen/used this before but am glad you're making use of it in the Python3.8 update!

Reply:
pretty trippy! I thought we had some in golang here for a moment :)

Reply:
right?? I really wanted the excuse to use this syntax, so I was very happy when I found the chance haha!